Automattic/woocommerce-services
# stripe account setup (deferred + oauth)

This PR includes some basic UI to test the creation of Stripe deferred accounts and connecting a Stripe account via OAuth.

To Test:
- Checkout the stripe PR on the server side https://github.com/Automattic/woocommerce-connect-server/pull/960
- Install the Stripe gateway WooCommerce plugin
- Go to any admin page and there should be an admin notice to create/connect
